### PR TITLE
[W5.7][W14-3]Ling Zhi Yu

### PR DESCRIPTION
**Pull request by lingzhiyu**
Added count feature, with io tests, followed coding standards & OOP, updated user guide.
Ready for review.

### CODE REVIEW

**Comment by jeffypie369** [comment]
[W5.7] Team component

Nice addition, except it was dirtied with other commits.
Create more branches in the future!

-Jefferson

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (kexiaowen) [inline src/seedu/addressbook/parser/Parser.java:14]: Imported classes should always be listed explicitly. This is stated in our coding standard

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -3,8 +3,8 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.awt.*;
```
Comment (kexiaowen) [inline test/java/seedu/addressbook/common/UtilsTest.java:6]: same issue as above. Pls only import necessary classes

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -43,4 +43,16 @@ private void assertAreUnique(Object... objects) {
...
+    @Test
+    public void isAnyNull() throws Exception {
+        assertTrue(Utils.isAnyNull(null, null)); // Test for nulls
+        assertTrue(Utils.isAnyNull(1, null));
+        assertTrue(Utils.isAnyNull(new String[1]));
+        //assertFalse(Utils.isAnyNull(Arrays.asList((Object) null)));
+        assertFalse(Utils.isAnyNull(new String[0]));
+        assertFalse(Utils.isAnyNull(new List[0]));
+        assertFalse(Utils.isAnyNull(1, 2)); // Test for int values
+        assertFalse(Utils.isAnyNull("abn", "ab")); // Test for String values
```
Comment (kexiaowen) [inline test/java/seedu/addressbook/common/UtilsTest.java:56]: As Jefferson pointed out, this PR should not contain commits for last week's LO

**Review by kexiaowen — APPROVED** [review]
@omegafishy Good try Zhi Yu! You might also try to write unit tests for the count command. Pls close this PR after reading my comments.

**Pull request closed without merging**